cmd/kpod/parse.go

```diff
@@ -765,7 +766,7 @@ func parseUser(rootdir, userspec string) (specs.User, error) { //nolint
 		// We parsed the user name as a number, and there's no group
 		// component, so we need to look up the user's primary GID.
 		var name string
-		name, gid64, gerr = lookupGroupForUIDInContainer(rootdir, uid64)
+		name, gid64, gerr = chrootuser.LookupGroupForUIDInContainer(rootdir, uid64)
```

Comment:
We can't handle these in cmd/kpod, so I suppose we need to add a WithUser and/or WithGroup in libpod and do the lookups there